packages/cli/src/commands/devices.ts

```diff
@@ -51,6 +55,10 @@ export default class DevicesCommand extends ListingOutputAPICommand<Device, Devi
 			description: 'filter results by device',
 			multiple: true,
 		}),
+		verbose: flags.boolean({
+			description: 'include location name in table output',
```

Comment:
I would drop "in table output" here because this will also show up in json or yaml output.